ingest/ingestHelpers.py

```diff
@@ -161,27 +164,18 @@ def get_authors(ds):
 # get_distributions: object -> [distributions] for objects such as: datasets, publications, ...
 def get_distributions(ds):
     distributions = []
-    distributionList = [faux for faux in ds.objects(DCO.hasDistribution) if has_type(faux, DCAT.Distribution)]
+    distributionList = [faux for faux in ds.objects(DCAT.distribution) if has_type(faux, DCODATA.Distribution)]
     for distribution in distributionList:
-        accessURL = str(list(distribution.objects(DCO.accessURL))[0])
+
+        accessURL = list(distribution.objects(DCAT.accessURL))[0]
```

Comment:
What if there are no values for ``distribution.objects(DCAT.accessURL)``?  The reference to index 0 might return an index out of bounds error.